base/linalg/blas.jl

```diff
@@ -808,7 +869,8 @@ function trsv! end
 
 Returns the solution to `A*x = b` or one of the other two variants determined by `tA`
 (transpose `A`) and `ul` (triangle of `A` is used.) `dA` indicates if `A` is unit-triangular
-(the diagonal is assumed to be all ones).
+(the diagonal is assumed to be all ones if `dA` is `U`, and this is not assumed if
+`dA` is `N` (so that the diagonal values are read in)).
```

Comment:
instead of nested parens, how about "or the diagonal values are read from `A` if `dA` is `N`" ?